src/ray/protobuf/gcs_service.proto

```diff
@@ -557,6 +566,7 @@ message InternalKVKeysReply {
 // Service for KV storage
 service InternalKVGcsService {
   rpc InternalKVGet(InternalKVGetRequest) returns (InternalKVGetReply);
+  rpc InternalKVMultiGet(InternalKVMultiGetRequest) returns (InternalKVMultiGetReply);
```

Comment:
Can you add docstring to explain how it is different from `InternalKVGet `? Especially if we fail to obtain one of keys, what's going to happen?